.readthedocs.yaml

```diff
@@ -3,18 +3,16 @@
 version: 2
 
 build:
-  image: latest
+  os: ubuntu-22.04
+  tools:
+    python: 3
```

Comment:
@JKatzwinkel this could be more specific, but the schnöde 3 is simple enough, innit?

Reply:
why not build on ubuntu-latest?

Reply:
it's not available: https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os

Reply:
interesting